docs/modules/ROOT/pages/ios_task_scheduling.adoc

```diff
@@ -0,0 +1,33 @@
+= Task Scheduling
+
+Background tasks are scheduled based on the app's current context, and that context is based on a combination of factors. 
+These factors include:
+
+* Is the app backgrounded or in the foreground?
+* Is WiFi available?
+* Is the device in Low Power Mode?
+* Is the device connected to external power?
+* Has the photo library has changed?
+
+== Tasks 
+
+Currently, two tasks are available:
+
+* *Instant photo uploads:* This task is triggered when:
+** The photo library has new assets
+** The user is connected to a WiFi network
+* *Bookmark update task:* This task is triggered by the background fetch event scheduled by iOS
+
+=== Instant Photo Upload
+
+There are some things to be aware of, regarding Instant Photo Upload.
+
+* Photo and video upload can be enabled and disabled separately.
+* Before instant media upload can begin, the account and upload path have to be selected
+* Changes in the photo library are detected when the app goes into the foreground.
```

Comment:
How does this line interact with "This task is triggered when: The photo library has new assets"
This line makes me think that the task is not going to get triggered until there are new photos AND the app goes into the foreground.

Reply:
Will review and update.

Reply:
@mneuwert, in #84, you wrote: 

> Changes in the photo library are detected when app goes into foreground (no signification location change detection magic as in some other apps to make it work in background). However, as soon as NSURLSession instances are created uploads will continue in the background.

Does this apply to videos as well?

Reply:
"Assets" in the "photo library" include videos as well.

Reply:
Thank you.